test/test.js

```diff
@@ -113,6 +113,12 @@ describe('hotkey', function () {
       document.dispatchEvent(new KeyboardEvent('keydown', {shiftKey: true, code: 'KeyB', key: 'B'}))
       assert.notInclude(elementsActivated, 'button1')
     })
+
+    it('supports comma as a hotkey', function () {
+      setHTML('<button id="button1" data-hotkey="comma">Button 1</button>')
```

Comment:
This is great to see! Can you please also add a test for the comma in concert with some modifier keys such as `Control`?

Reply:
Done